gen/definitions/generic/url_filtering_definition.yaml

```diff
@@ -0,0 +1,88 @@
+---
+name: URL Filtering Policy Definition
+rest_endpoint: /template/policy/definition/urlfiltering/
+has_version: true
+id_attribute: definitionId
+doc_category: Security Policies
+attributes:
+  - model_name: type
+    type: String
+    value: urlFiltering
+  - model_name: name
+    tf_name: name
+    type: String
+    mandatory: true
+    description: The name of the policy definition.
+    example: Example
+  - model_name: description
+    tf_name: description
+    type: String
+    mandatory: true
+    description: The description of the policy definition.
+    example: My description
+  - model_name: mode
+    tf_name: mode
+    type: String
+    enum_values: [security, unified]
+    default_value: 
+    description: The policy mode
+    example: security
+  - model_name: alerts
+    data_path: [definition]
+    tf_name: alerts
+    type: StringList
+    description: List of alerts options that will be exported as syslog messages
+    example: blacklist
+  - model_name: webCategories
+    data_path: [definition]
+    tf_name: web_categories
+    type: StringList
+    description: List of categories to block or allow
+    example: alcohol-and-tobacco
+  - model_name: webCategoriesAction
+    data_path: [definition]
+    tf_name: web_categories_action
+    type: String
+    enum_values: [block, allow]
+    description: whether the selected web categories should be blocked or allowed.
+    example: allow
+  - model_name: webReputation
+    data_path: [definition]
+    tf_name: web_reputation
+    type: String
+    enum_values: [high-risk, suspicious, moderate-risk, low-risk, trustworthy]
+    description: The web reputation of the policy definition
+    example: moderate-risk
+  - model_name: targetVpns
+    data_path: [definition]
+    tf_name: target_vpns
+    type: StringList
+    description: List of VPN IDs
+    example: 1
+  - model_name: ref
+    data_path: [definition, urlWhiteList]
+    tf_name: url_white_list_id
```

Comment:
The terminology being used in the UI and also in this provider (https://registry.terraform.io/providers/CiscoDevNet/sdwan/latest/docs/resources/allow_url_list_policy_object) is `allow_url_list`, so I would suggest to rename this to `allow_url_list_id`.